### PR TITLE
refactor(crud and models)

### PR DESCRIPTION
**Pull request by ChrOertlin**
### Added

Refactors the crud of the database and moves models into a more appropriate space.

### Fixed

**Steps to consider while deploying**
- Configuration changes:
- Documentation updates:
- Inform users by email:

### Review:
- [ ] Code approved by
- [ ] Tests executed on stage by:  (Document the test done with screen shots and description.)
- [ ] "Merge and deploy" approved by

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: genotype_api/api/endpoints/samples.py [hunk 1]
```diff
@@ -1,33 +1,34 @@
...
-    Sample,
-    SampleReadWithAnalysis,
-    SampleRead,
-    User,
     SampleDetail,
-    Analysis,
     MatchResult,
     MatchCounts,
+)
+from genotype_api.database.models.models import (
```
Comment (henrikstranneheim) [inline genotype_api/api/endpoints/samples.py:16]:
```suggestion
from genotype_api.database.models import (
```
Or do we need the extra level?

**Review by henrikstranneheim — APPROVED** [review]
💯

**Comment by ChrOertlin** [comment]
Tested on stage
image

**Comment by ChrOertlin** [comment]
Ignoring sonarcloud - pionts to a model as being duplicated, names differ though

**Pull request merged**